### PR TITLE
New Keyboard: Booster

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Introducing a new numpad from Percent Studio called the booster. As Percent starts to get more boards in I figured it's best to start putting it in a Percent directory. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/percent/readme.h [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Percent Studio
```
Comment (fauxpark) [inline keyboards/percent/readme.h]: This should be an .md file ;)

File: keyboards/percent/booster/info.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            "key_count": 17,
+            "layout": [
+                {"label":"K00 (D1,C7)", "x":0, "y":0},
+                {"label":"K01 (D1,D4)", "x":1, "y":0},
+                {"label":"K02 (D1,D2)", "x":2, "y":0},
+                {"label":"K03 (D1,D0)", "x":3, "y":0},
+                {"label":"K10 (D6,C7)", "x":0, "y":1},
+                {"label":"K11 (D6,D4)", "x":1, "y":1},
+                {"label":"K12 (D6,D2)", "x":2, "y":1},
+                {"label":"K23 (D7,D0)", "x":3, "y":1, "h":2},
```
Comment (noroadsleft) [inline keyboards/percent/booster/info.json]: This line needs to move down...

```suggestion
```

File: keyboards/percent/booster/info.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                {"label":"K20 (D7,C7)", "x":0, "y":2},
+                {"label":"K21 (D7,D4)", "x":1, "y":2},
+                {"label":"K22 (D7,D2)", "x":2, "y":2},
```
Comment (noroadsleft) [inline keyboards/percent/booster/info.json]: ... to here.

```suggestion
                {"label":"K22 (D7,D2)", "x":2, "y":2},
                {"label":"K23 (D7,D0)", "x":3, "y":1, "h":2},
```

File: keyboards/percent/booster/info.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+                {"label":"K30 (B4,C7)", "x":0, "y":3},
+                {"label":"K31 (B4,D4)", "x":1, "y":3},
+                {"label":"K32 (B4,D2)", "x":2, "y":3},
+                {"label":"K33 (B4,D0)", "x":3, "y":3, "h":2},
```
Comment (noroadsleft) [inline keyboards/percent/booster/info.json]: And this line...

```suggestion
```

File: keyboards/percent/booster/info.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+                {"label":"K40 (B5,C7)", "x":0, "y":4, "w":2},
+                {"label":"K42 (B5,D2)", "x":2, "y":4}
```
Comment (noroadsleft) [inline keyboards/percent/booster/info.json]: ... needs to move here.

```suggestion
                {"label":"K42 (B5,D2)", "x":2, "y":4},
                {"label":"K33 (B4,D0)", "x":3, "y":3, "h":2}
```

File: keyboards/percent/booster/info.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+            ]
+        }
+    }
+    ,"meta": "https://noroadsleft.github.io/kbf_qmk_converter/"
```
Comment (noroadsleft) [inline keyboards/percent/booster/info.json]: And you can take this out now that I know where this code came from. :)

```suggestion
```

File: keyboards/percent/booster/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,27 @@
+/* Copyright 2019 REPLACE_WITH_YOUR_NAME
```
Comment (noroadsleft) [inline keyboards/percent/booster/keymaps/default/keymap.c]: :)

```suggestion
/* Copyright 2019 MechMerlin
```

File: keyboards/percent/booster/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_numpad_5x4(
+        KC_NLCK, KC_SLSH, KC_ASTR, KC_MINS,
```
Comment (noroadsleft) [inline keyboards/percent/booster/keymaps/default/keymap.c]: Looks like a kbfirmware bug of sorts here; these aren't numpad keycodes.

```suggestion
        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
```

File: keyboards/percent/booster/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+        KC_P7,   KC_P8,   KC_P9,   KC_PLUS,
```
Comment (noroadsleft) [inline keyboards/percent/booster/keymaps/default/keymap.c]: Plus key needs to move down a row.

```suggestion
        KC_P7,   KC_P8,   KC_P9,
```

File: keyboards/percent/booster/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+        KC_P4,   KC_P5,   KC_P6,
```
Comment (noroadsleft) [inline keyboards/percent/booster/keymaps/default/keymap.c]:
```suggestion
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
```

File: keyboards/percent/booster/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+        KC_P1,   KC_P2,   KC_P3,   KC_ENT,
```
Comment (noroadsleft) [inline keyboards/percent/booster/keymaps/default/keymap.c]: Enter needs to move down a row too.

```suggestion
        KC_P1,   KC_P2,   KC_P3,
```

File: keyboards/percent/booster/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+        KC_P0,            KC_PDOT
```
Comment (noroadsleft) [inline keyboards/percent/booster/keymaps/default/keymap.c]:
```suggestion
        KC_P0,            KC_PDOT, KC_PENT
```

**Comment by drashna** [comment]
Thanks

**Pull request merged**